src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnection.cs

```diff
@@ -40,6 +40,9 @@ public sealed partial class SqlConnection : DbConnection
         internal bool _supressStateChangeForReconnection;
         private int _reconnectCount;
 
+        // diagnostics listener
+        private readonly static DiagnosticListener _diagnosticListener = new DiagnosticListener(SqlClientDiagnosticListenerExtensions.DiagnosticListenerName);
```

Comment:
@saurabh500 I have changed the name to `s_diagnosticListener` and this change can be found in commit https://github.com/dotnet/corefx/pull/8420/commits/319736ba1954415602e7aecf25ba52daee275f36 Thanks! :smile: